--- Stikked-0.10.0/config/stikked.php~	2015-09-30 15:25:59.000000000 +0300
+++ Stikked-0.10.0/config/stikked.php	2016-08-16 13:07:37.524958709 +0300
@@ -15,10 +15,10 @@
  * The database structure will be created automatically
  *
 */
-$config['db_hostname'] = '127.0.0.1';
+$config['db_hostname'] = 'localhost';
 $config['db_database'] = 'stikked';
-$config['db_username'] = 'stikked';
-$config['db_password'] = 'stikked';
+$config['db_username'] = 'mysql';
+$config['db_password'] = '';
 //$config['db_driver'] = 'sqlite';
 
 /**
--- Stikked-0.10.0/htdocs/index.php~	2016-08-16 13:07:05.000000000 +0300
+++ Stikked-0.10.0/htdocs/index.php	2016-08-16 13:09:35.977792314 +0300
@@ -53,7 +53,7 @@
  *
  * NOTE: If you change these, also change the error_reporting() code below
  */
-	define('ENVIRONMENT', isset($_SERVER['CI_ENV']) ? $_SERVER['CI_ENV'] : 'development');
+	define('ENVIRONMENT', isset($_SERVER['CI_ENV']) ? $_SERVER['CI_ENV'] : 'production');
 
 /*
  *---------------------------------------------------------------
@@ -72,7 +72,6 @@
 
 	case 'testing':
 	case 'production':
-		ini_set('display_errors', 0);
 		if (version_compare(PHP_VERSION, '5.3', '>='))
 		{
 			error_reporting(E_ALL & ~E_NOTICE & ~E_DEPRECATED & ~E_STRICT & ~E_USER_NOTICE & ~E_USER_DEPRECATED);
