--- Stikked-0.10.0/./htdocs/themes/default/views/view/qr.php~	2016-08-16 13:10:51.000000000 +0300
+++ Stikked-0.10.0/./htdocs/themes/default/views/view/qr.php	2016-08-16 13:11:10.406041808 +0300
@@ -4,7 +4,7 @@
  * Function list:
  * Classes list:
  */
-require_once (APPPATH.'/libraries/phpqrcode/qrlib.php');
+require_once '/usr/share/php/phpqrcode/phpqrcode.php';
 $qrurl = $url;
 
 if ($snipurl != false) 
